crates/router/src/routes/user.rs

```diff
@@ -575,7 +575,7 @@ pub async fn verify_recon_token(state: web::Data<AppState>, http_req: HttpReques
         &http_req,
         (),
         |state, user, _req, _| user_core::verify_token(state, user),
-        &auth::ReconJWT,
+        &auth::DashboardNoPermissionAuth,
```

Comment:
This can also use JWT with Recon Admin permission.

Reply:
This API is used by recon, they don't have recon admin API key

Reply:
But the JWT will have permissions for Recon right.